rotkehlchen/exchanges/exchange.py

```diff
@@ -214,10 +214,10 @@ def query_online_deposits_withdrawals(
         """Queries the exchange's API for the asset movements of the user
 
         Should be implemented in subclasses.
+
+        Deprecated, will be replaced by query_online_history_events
```

Comment:
Why do you want to deprecate this?

Some exchanges query "all transactions".
Some exchanges have different endpoints for each type such as trade, deposit/withdrawal etc.

This function is eventually called by the frontend at asset movements view to refresh the user's asset movements. So by removing that frontend view you also think we should conslidate those in the exchange interface?

Reply:
> So by removing that frontend view you also think we should conslidate those in the exchange interface?

Yes, I assumed that since that view is going away there would be no point in using different functions for AssetMovements versus other history events, since they will all need queried at the same time anyway. And this function can be used for other things we may convert to history events in the future as well, such as Trades.

Reply:
As I wrote in the other comment, the view may go away but the way we query stuff does not mean that it will be so condensed as one api query for all history events. You need to have the ability to be a bit more flexible.

For example you would need to be able to at least refresh/query latest events of a single exchange at least. When you go to the exchange specific view, or when you add a new api key for that exchange for example.

Reply:
As mentioned here: https://github.com/rotki/rotki/pull/9042#discussion_r1878299682 I've added an endpoint for querying latest events from a single exchange.